osdk/src/config/mod.rs

```diff
@@ -61,7 +69,10 @@ fn apply_args_before_finalize(action_scheme: &mut ActionScheme, args: &CommonArg
             }
         }
         if let Some(initramfs) = &args.initramfs {
-            boot.initramfs = Some(initramfs.clone());
+            let initramfs = initramfs
+                .canonicalize()
+                .expect("initramfs from cli is invalid");
```

Comment:
Use `error_msg!` and `process::exit` to report errors to the user. Only panic in a bad program state, which could only happen if there's a bug in OSDK.